tools/distpartitioning/convert_partition.py

```diff
@@ -129,7 +130,9 @@ def create_dgl_object(graph_name, num_parts, \
     assert len(uniq_ids) == len(idx)
     # We get the edge list with their node IDs mapped to a contiguous ID range.
     part_local_src_id, part_local_dst_id = np.split(inverse_idx[:len(shuffle_global_src_id) * 2], 2)
-    compact_g = dgl.graph((part_local_src_id, part_local_dst_id))
+
+    #compact_g = dgl.graph((part_local_src_id, part_local_dst_id))
+    compact_g = dgl.graph(data=(part_local_src_id, part_local_dst_id), num_nodes=len(idx))
```

Comment:
is this a bug fix to the problem you encountered?

Reply:
Yes. This is a bug fix for the mag_lsc dataset. In testing this dataset, we run into a peculiar situation where in one of the partitions the unique nodes in the edge list does not completely represent all the nodes in that partition. By using the no. of nodes, this issue has been resolved.